docs/IR.md

```diff
@@ -370,7 +370,9 @@ Which is referenced by the Tensor type message:
   }
 ```
 
-The empty list of dimension sizes, [], is a valid tensor shape, denoting a zero-dimension (scalar) value. A zero-dimension tensor is distinct from a tensor of unknown dimensionality, which is indicated by an absent 'shape' property in the Tensor record. When the shape property is absent for an input, a tensor value of any shape may be passed from the caller. When the shape property is absent for an output, the caller should expect that the output value may be of any shape.
+The empty list of dimension sizes, [], is a valid tensor shape, denoting a zero-dimension (scalar) value. A zero-dimension tensor is distinct from a tensor of unknown dimensionality, which is indicated by an absent 'shape' property in the Tensor record. When the shape property is absent for a node input, a tensor value of any shape may be passed from the caller. When the shape property is absent for a node output, the caller should expect that the output value may be of any shape. 
```

Comment:
Specify:
input -> node input
output -> node output

Reply:
I suggest changing the "When the shape property is absent ..." part as below:
```
When the shape property is absent in the type of a value, it indicates that the corresponding runtime value
may have any shape. However, types with a missing shape are not permitted in all contexts. In particular,
the inputs and outputs of a model are required to *have* a shape, indicating the rank of inputs and outputs,
even though the exact dimensions need not be specified.
```